### ISSUE TITLE
AttributeError: 'HTTPResponse' object has no attribute 'strict'

### ISSUE THREAD

**Issue by Rodrigo-Novas** [urllib3/urllib3#3054]
### Subject

From what I understand the error comes because the [cachecontrol](https://pypi.org/project/CacheControl/#description) dependency (which is used by many libraries) uses the strict parameter, but in the latest version of urrlib3 that parameter is no longer there.

I saw that you wrote a [warning](https://github.com/urllib3/urllib3/blob/main/src/urllib3/poolmanager.py#L315) for 2.1.0 version, but even so the strict is already removed.

This happens to me when installing dependencies with poetry Ex: `poetry install`

### Environment

This error occurred to me using ubuntu and macOs M1 with python >3.8.
Installing only Poetry 1.1.6.

At least, paste here the output of:

```
AttributeError

  'HTTPResponse' object has no attribute 'strict'
```

### Steps to Reproduce

- Use ubuntu or MacOs M1
- Install poetry and make sure you have urllib3 version >=2. 
- Run the poetry install command

### Expected Behavior

The strict parameter must not be removed.

### Actual Behavior

The strict parameter was removed.

**The error does not occur if I use the urrlib3<2 constraint**

**Comment by sigmavirus24** [comment]
Duplicate of #3010

**Comment by Rodrigo-Novas** [comment]
Idk that was duplicated i will close it, but.. ¿Is there an update about this?

**Comment by pquentin** [comment]
Can you please try cachecontrol 0.13.0 that was just released? It should be compatible with urllib3 2.0. In any case, closing this issue in favor of the other one.

**Comment by Rodrigo-Novas** [comment]
Yes! with cachecontrol 0.13.0 works fine!
